### DISCUSSION TITLE
Welcome to yadoms Discussions!

### DISCUSSION THREAD

**Discussion by jano42** [Yadoms/yadoms#714] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gotgot1995** [comment]
Hi @jano42 , I'm Gautier and I'm a french software engineer. I'd like to use Yadoms to build my own smart home and contribute to the project by developing plugins and such. 🙂 I want to join the community to share my ideas and also ask the community for suggestions in order to implement new features.

**Comment by jano42** [comment]
Hi @gotgot1995 ! we welcome new recruits with pleasure ! We will contact you asap
